x-pack/plugins/maps/server/data_indexing/get_indexes_matching_pattern.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IScopedClusterClient } from 'kibana/server';
+import { CatIndicesRecord } from '@elastic/elasticsearch/api/types';
+import { MatchingIndexesResp } from '../../common';
+
+export async function getMatchingIndexes(
```

Comment:
Can you add an API integration test for this endpoint at https://github.com/elastic/kibana/tree/master/x-pack/test/api_integration/apis/maps